unlock-app/src/services/storageService.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+export default class StorageService {
+  HOST = 'http://localhost:4000'
```

Comment:
Will definitely be removed and replaced with proper configuration aware lookup

Reply:
+ security ;)